run

```diff
@@ -509,6 +509,7 @@ lint() {
             --exclude build-\* \
             --exclude install-\* \
             --exclude applications/holoviz/template/cookiecutter\* \
+            --exclude operators/grpc_operators/generated\* \
```

Comment:
Would be good to remove in favor of outputting generated files to the binary directory